### ISSUE TITLE
Scroll pages with j/k immediate start

### ISSUE THREAD

**Issue by SamuelFe** [qutebrowser/qutebrowser#6845]
A minor issue but I need to point it out.

At the pressing of `j` or `k` for scrolling the behavior is: a little movement happens at first and only after some delay the page starts to scroll.

I would like to the pages to immediately start scrolling *up/down* as I press `j` or `k`. Or to have the option to do that at least.

This is a minor issue since I just *think* that way to go would be great. A behavior like it is right now must be thought to be like that (don't really know), so maybe the behavior I want was already considered and had some problem I didn't considered...

**Comment by The-Compiler** [comment]
Duplicate of #4768
